api/swagger-spec/oapi-v1.json

```diff
@@ -20355,17 +20373,6 @@
      "forcePull": {
       "type": "boolean",
       "description": "forcePull describes if the builder should pull the images from registry prior to building."
-     },
-     "runtimeImage": {
-      "$ref": "v1.ObjectReference",
```

Comment:
@php-coder don't forget when you have time to add this back so i don't fetch this :dancers: